src/utils/theme.js

```diff
@@ -39,5 +40,26 @@ const theme = createTheme({
 
 export default theme
 
+const seasTheme = createTheme({
+  primary: {
+    main: '#5094FB',
+    contrastText: '#fff',
+    background: 'rgba(157, 75, 163, 0.08)',
+  },
+  secondary: {
+    main: '#29BEBA',
+    contrastText: '#fff',
+  },
+})
+
 export const extendTheme = (primaryTheme, extendedTheme) =>
   createTheme(merge({}, primaryTheme, extendedTheme))
+
+export const themeMapper = (causeId) => {
```

Comment:
Rather than map on cause ID, which still pushes a lot of logic to the client, let's use the actual theme values we want to change: `{ primaryColor, secondaryColor }` etc. Those should live in the Cause model "theme" property—which we didn't spec.

As a shortcut, we can still map ID to values that we pass into this hook, but this sets us up better for generalizability.